R/tuneParams.R

```diff
@@ -151,12 +151,16 @@ tuneParams <- function(params,
         p <- prepare_params(p)
     }
 
+    ##RF How many backgrounds are being used in the params object? Section not use for now, don;t know how to carry no_resource around
```

Comment:
I think these 4 lines should not be here any more, right?

Reply:
Completely missed them sorry ^^'